client/my-sites/jetpack-search/purchases.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import { isJetpackSearch, planHasJetpackSearch } from '@automattic/calypso-products';
+
+export const hasJetpackSearchPurchaseOrPlan = (
+	sitePurchases: Record< string, unknown > | null,
```

Comment:
`sitePurchase` looks like an `Array` to me

Reply:
Good spot 👍 I'll fix that.